pkg/descheduler/descheduler.go

```diff
@@ -97,7 +97,7 @@ func RunDeschedulerStrategies(ctx context.Context, rs *options.DeschedulerServer
 	wait.Until(func() {
 		nodes, err := nodeutil.ReadyNodes(ctx, rs.Client, nodeInformer, nodeSelector, stopChannel)
 		if err != nil {
-			klog.V(1).Infof("Unable to get ready nodes: %v", err)
+			klog.ErrorS(err, "Unable to get ready nodes")
```

Comment:
I think we want to use `InfoS` here. I know we went back and forth a few times as part of the review, but I believe we want to use `InfoS` here. Hopefully the final answer. :-)

Reply:
done